src/bot.py

```diff
@@ -116,6 +115,9 @@ def bot():
             print_with_random_help_msg()
         elif command == "add-birthday":
             print(add_birthday(args, book))
+            print_with_random_help_msg()   
+        elif command == "change-birthday":
```

Comment:
Please add this command to commands list for autocompletion otherwise this command will not work